common.js

```diff
@@ -2938,6 +2938,71 @@ const allSoloCars = {
       'all' : ['ss', 'ssm', 'ssp'],
     }
   },
+  'Geo':{
+    'Metro':{
+      'all': ['hs']
+    },
+    'Prizm':{
+      'all': ['hs']
+    },
+    'Spectrum':{
+      'all': ['hs']
+    },
+    'Storm':{
+      '2005': ['sp'],
+      '2006': ['dsp'],
+      '2007': ['dsp'],
+    },
+    'Cobalt SS (N/A)':{
```

Comment:
The Cobalt and Cobalt SS are Chevrolet models.

Reply:
Sorry I misread the pdf